src/Components/DrillPage.js

```diff
@@ -51,10 +51,31 @@ export const DrillPage = (props) => {
   const share = async (drill) => {
     const url = Linking.makeUrl('drills/' + drill.id);
 
+    //liens Youtube
+    let youtubeLinks = [];
+
+    for (let i = 0; i < drill.steps.length; i++) {
+      if (drill.steps[i].youtube) {
+        let dict = new Object();
+        dict['title'] = drill.steps[i].title;
+        dict['youtubeUrl'] = drill.steps[i].youtube;
+        youtubeLinks.push(dict);
```

Comment:
```suggestion
        youtubeLinks.push({
          title: drill.steps[i].title,
          youtubeUrl: drill.steps[i].youtube,
        });
```
On n'a pas besoin d'instancier un objet en js avant de le remplir, tu peux direct le créer et l'utiliser